tests/test_api/test_util.py

```diff
@@ -17,7 +17,7 @@ def test_get_granules():
             {'job_parameters': {'granules': ['F', 'F']}},
```

Comment:
Can we add `reference` and `secondary` here as well so that we're testing the new behavior?